service/ui.go

```diff
@@ -89,16 +89,15 @@ func NewUIServer(env evergreen.Environment, queue amboy.Queue, home string, fo T
 	}
 
 	uis := &UIServer{
-		Settings:           *settings,
-		env:                env,
-		queue:              queue,
-		Home:               home,
-		clientConfig:       evergreen.GetEnvironment().ClientConfig(),
-		CookieStore:        sessions.NewCookieStore([]byte(settings.Ui.Secret)),
-		buildBaronProjects: plugin.BbGetConfig(settings),
```

Comment:
Why did you get rid of this? I'm not sure we can do this.

Reply:
I removed buildBaronProjects from the UIServer because it was not referenced anywhere in the codebase, and we will not need this to live in UIServer moving forward.